### PR TITLE
Add actuallymentor/battery to homebrew cask

### PR DESCRIPTION
**Pull request by actuallymentor**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/battery.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "battery" do
+  version "1.0.3"
+  sha256 "71e699bcf634f481a0b1c54a2035a04e9e9fec4463c9dbfe95d0370a89aaaabc"
+
+  url "https://github.com/actuallymentor/battery/releases/download/v#{version}/battery-#{version}-arm64.dmg"
```
Comment (SMillerDev) [inline Casks/battery.rb:5]: Is there no x86?

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/battery.rb (hunk 1 above)
Comment (miccal) [reply Casks/battery.rb:5]: No, this is for Arm only.

**Comment by actuallymentor** [comment]
> Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

@SMillerDev added a zap stanza as requested! I haven't created one before so let me know if I understood correctly to add any paths the installation modifies.

File: Casks/battery.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  license "MIT"
+  zap trash: [
+    "~/.battery",
+    "~/Library/Application Support/battery",
+    "~/Library/LaunchAgents/battery.plist",
+    "~/Library/Preferences/co.palokaj.battery.plist",
+    "~/Library/Preferences/org.mentor.Battery.plist",
+    "~/Library/Saved Application State/co.palokaj.battery.savedState",
+    "/usr/local/bin/smc",
```
Comment (SMillerDev) [inline Casks/battery.rb:22]: I think this should be in `uninstall delete:`

Comment (actuallymentor) [reply Casks/battery.rb:22]: Ah I see. I amended that section. Let me know if I understood correctly.

Comment (bevanjkay) [reply Casks/battery.rb:22]: @actuallymentor I believe that the `uninstall delete:` specifically relates to the `/usr/local/bin/smc` - the rest were in the right place.

**Comment by miccal** [comment]
Thank you @actuallymentor.

**Review by miccal — APPROVED** [review]

**Pull request merged**